.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,10 @@
-<!--
-Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+### What / Why
 
-This PR template was copied from https://github.com/testing-library/react-testing-library/blob/main/.github/PULL_REQUEST_TEMPLATE.md
 
-Please fill out the information below to expedite the review and (hopefully)
-merge of your pull request!
--->
+### How
```

Comment:
How it fixes it? (A technical description of what have changed)

Reply:
yes. The non-technical description of the fix should go under the `What / Why` heading

Reply:
I was meaning to add it to the template.

Reply:
done